recipes/libidn2/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set version = "2.0.4" %}
+
+package:
+  name: libidn2
+  version: "{{ version }}"
+
+source:
+  url: https://ftp.gnu.org/gnu/libidn/libidn2-{{ version }}.tar.gz
+  sha256: "644b6b03b285fb0ace02d241d59483d98bc462729d8bb3608d5cad5532f3d2f0"
+
+build:
+  number: 0
+  run_exports:
+    # _Amazing_ backwards compatibility so pin to major number:
+    #    https://abi-laboratory.pro/tracker/timeline/libidn2/
+    - {{ pin_subpackage('libidn2', min_pin='x', max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - test -f "${PREFIX}/include/idn2.h"
+    - test -f "${PREFIX}/lib/libidn2.so"     # [linux]
+    - test -f "${PREFIX}/lib/libidn2.dylib"  # [osx]
+    - conda inspect linkages -p "$PREFIX" "$PKG_NAME"
```

```suggestion
    - conda inspect linkages -p "$PREFIX" "$PKG_NAME"  # [unix]
```

AFAIK works only on mac an linux